boards/arm/nucleo_l152re/nucleo_l152re.dts

```diff
@@ -39,6 +39,8 @@
 		led0 = &green_led_0;
 		sw0 = &user_button;
 		eeprom-0 = &eeprom;
+		adc1 = &adc1;
+		dac1 = &dac1;
```

Comment:
I think this is redundant as sample does not force aliases. Could just use `DT_NODELABEL` as other boards do in the sample.

Reply:
I see. Aliases were the old way of doing things. I'll clean this up for both Nucleo boards by a separate commit.

Reply:
Done.